projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_grid.scss

```diff
@@ -221,6 +222,10 @@ $_light-grid: extend(
             igx-color: ('grays', 600)
         ),
 
+        cell-disabled-color: (
+            igx-color: ('grays', 400)
```

Comment:
Where did you get this color from?

Reply:
This looked closest to the current design in the row pinning spec. You are free to correct me if this is not right.

Reply:
The gray variant that we use for disabled is 500.

Reply:
should be 500